pkg/discovery/dtofactory/workload_controller_entity_dto_builder.go

```diff
@@ -137,6 +137,14 @@ func (builder *workloadControllerDTOBuilder) getCommoditiesBought(kubeController
 func (builder *workloadControllerDTOBuilder) getContainerSpecIds(kubeController *repository.KubeController) []string {
 	containerNameSet := make(map[string]struct{})
 	for _, pod := range kubeController.Pods {
+		// Pods scheduled on a node, but still in Pending phase do not have containers started, exclude them
+		// when getting container specs
+		if discoveryUtil.PodIsPending(pod) {
+			glog.V(3).Infof("Skipping pod %v/%v when building containerSpecs owned by controller %v/%v."+
+				" There is no container started in the pod.",
+				pod.Namespace, pod.Name, kubeController.Namespace, kubeController.Name)
```

Comment:
Nit: we have some existing functions to show the proper pod/controller name with namespace something like:
```
glog.V(3).Infof("Skipping pod %v when building containerSpecs owned by controller %v."+
				" There is no container started in the pod.",
				discoveryUtil.GetPodClusterID(pod), kubeController.GetFullName())

```

Reply:
Changed accordingly. Now we see the following logs:
```
I0309 18:41:58.056181   43802 workload_controller_entity_dto_builder.go:143] Skipping pod ccp/cert-manager-7c4fdf69b7-bc67h when building containerSpecs owned by controller ccp/Deployment/cert-manager. There is no container started in the pod.
I0309 18:41:58.056294   43802 workload_controller_entity_dto_builder.go:143] Skipping pod kube-system/coredns-7c4d89dd66-jr5zj when building containerSpecs owned by controller kube-system/Deployment/coredns. There is no container started in the pod.
```